addon/components/paper-button.js

```diff
@@ -4,45 +4,36 @@ import RippleMixin from '../mixins/ripple-mixin';
 import ProxiableMixin from 'ember-paper/mixins/proxiable-mixin';
 import ColorMixin from 'ember-paper/mixins/color-mixin';
 
+const {
+  assert,
+  computed,
+  isPresent
+} = Ember;
+
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
-  attributeBindings: ['target', 'action', 'type'],
+  attributeBindings: ['type'],
   tagName: 'button',
+  classNames: ['paper-button', 'md-default-theme'],
   themed: true,
-  classNameBindings: ['raised:md-raised', 'icon-button:md-icon-button', 'focus:md-focused', 'themed:md-default-theme', 'themed:md-button'],
+  classNameBindings: [
+    'raised:md-raised',
+    'iconButton:md-icon-button',
+    'themed:md-default-theme',
+    'themed:md-button'
```

Comment:
these classes should be hardcoded into `classNames` directly.
Remove `themed` property.